sklearn/utils/weight_vector.pyx

```diff
@@ -30,9 +30,9 @@ cdef class WeightVector(object):
 
     Attributes
     ----------
-    w : ndarray, dtype=double, order='C'
+    w : ndarray, dtype=double
```

Comment:
These are views now. I don't know how strict we want to be with the docstrings, I guess that's fine.

Reply:
There's a mix between parameters and attributes. w and aw are both.
As an attribute they are memoryviews but as parameters they are objects that implement the array interface (e.g. numpy arrays)